### PR TITLE
wait 120 seconds for CAL

### PR DESCRIPTION
**Pull request by capri-xiyue**
Fixes #

## Proposed Changes

- wait 120 seconds for CAL sources in E2E 
-
-

**Release Note**

```release-note

```

**Docs**

### CODE REVIEW

File: test/e2e/lib/resources/constants.go [hunk 1]
```diff
@@ -60,3 +60,8 @@ const (
 const (
 	WaitDeletionTime = 20 * time.Second
 )
+
+// WaitCALTime for time needed to wait to fire an event after CAL Source is ready
+const (
+	WaitCALTime = 120 * time.Second
```
Comment (nachocano) [inline test/e2e/lib/resources/constants.go:66]: nit: putting under the same const block as WaitDeletionTime

File: test/e2e/test_auditlogs.go [hunk 2]
```diff
@@ -127,7 +127,8 @@ func CloudAuditLogsSourceWithTargetTestImpl(t *testing.T, authConfig lib.AuthCon
 
 	// Audit logs source misses the topic which gets created shortly after the source becomes ready. Need to wait for a few seconds.
 	// Tried with 45 seconds but the test has been quite flaky.
-	time.Sleep(90 * time.Second)
+	// Tried with 90 seconds but the test has been quite flaky.
```
Comment (nachocano) [inline test/e2e/test_auditlogs.go:130]: nit: move this comment where you define the 120 constant

**Comment by capri-xiyue** [comment]
/hold
Test the pipeline to see whether it can reduce the flakiness. This is just a workaround.

**Pull request merged**